doc/bib/elephant.bib

```diff
@@ -1,4 +1,23 @@
-@article{Gruen02_43, 
+@misc{ding2006granger,
```

Comment:
We should find a common bibtex key schema, I'd suggest [FirstAuthorLastName][Year2Digit]_[FirstPage]

Reply:
I have previously used [firstauthorlastname][year4digit][firstwordoftitle] (e.g. ding2006granger), but it would be fine either way.